website/404.html

```diff
@@ -0,0 +1,38 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "_includes/base-resp.html" %}
+
+{% block page_title_prefix %}{{_('Thunderbird')}} — {% endblock %}
+{% block page_title %}{{ _('There\'s nothing here!') }}{% endblock %}
+{% block page_desc %}{{ _('The page you are looking for doesn\'t exist. If you think this is a bug, you can <a href="https://github.com/thundernest/thunderbird-website">report an issue here</a>.') }}{% endblock %}
+
+{% block body_id %}thunderbird-landing{% endblock %}
```

Comment:
I don't see any reason to reuse the landing page body id here, and it could cause problems in the future if CSS styles are targeted at the landing page specifically. Make it unique.

```suggestion
{% block body_id %}thunderbird-404{% endblock %}
```

Reply:
Makes sense.